Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -8,8 +11,12 @@ import MapboxDirections
  Unlike `Router` classes such as `RouteController` and `LegacyRouteController`, this class operates without a predefined route, matching the user’s location to the road network at large. You can use a passive location manager to determine a starting point for a route that you calculate using the `Directions.calculate(_:completionHandler:)` method. If the user happens to be moving while you calculate the route, the passive location manager makes it less likely that the route will begin with a short segment on a side road or driveway and a confusing instruction to turn onto the current road.
  
  To find out when the user’s location changes, implement the `PassiveLocationManagerDelegate` protocol, or observe `Notification.Name.passiveLocationManagerDidUpdate` notifications for more detailed information.
+
+ - important: Creating an instance of this type will start a Free Drive session which can be paused by `PassiveLocationManager.pauseTripSession()` if you don't need navigation updates temporarily. The trip session is stopped when the instance is deallocated. From more info read the [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
```

Comment:
Some minor rewording to flow better; the other documentation comments below could use similar tweaks. We should also propagate these changes to the changelog and [migration guide](https://github.com/mapbox/mapbox-navigation-ios/wiki/Upgrading-from-v1.x-to-v2.0.x).

```suggestion
 - important: Creating an instance of this class will start a free-driving session. If the application goes into the background or you temporarily stop needing location updates for any other reason, temporarily pause the trip session using the `PassiveLocationManager.pauseTripSession()` method to avoid unnecessary costs. The trip session also stops when the instance is deinitialized. For more information, see the “[Pricing](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/)” guide.
```

We’ll need to remember to update the link to [this guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/) once we move the beta site to the main site.

Reply:
Thanks for the edit, reads better!